applications/API/forms.py

```
@@ -0,0 +1,322 @@
+        widget=widgets.CheckboxInput(attrs={"class": "form-check-input"}),
+        help_text="Check if you want to remain anonymous",
+    )
+
+    review_text = CharField(
+        required=False,
+        min_length=70,
+        label="Review Text",
+        widget=widgets.Textarea(attrs={"class": "form-control", "rows": 3}),
+    )
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)